src/Microsoft.Health.Fhir.TemplateManagement/OCIFileManager.cs

```diff
@@ -72,7 +72,10 @@ public void PackOCIImage(bool ignoreBaseLayers = false)
 
         public async Task PushOCIImageAsync()
         {
-            await _orasClient.PushImageAsync(_overlayFS.WorkingImageLayerFolder);
+            var rawLayers = _overlayFS.ReadImageLayers();
+            var fileLayers = _overlayOperator.ExtractOCIFileLayers(rawLayers);
+            var sortedLayers = _overlayOperator.SortOCIFileLayersBySequenceNumber(fileLayers);
```

Comment:
Seems we can always extract file layers in sequence number order. Can we merge ExtractOCIFileLayers and SortOCIFileLayersBySequenceNumber into one function?

Reply:
SortBySequenceNumber function will be removed if using manifest next release version.